src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -147,6 +149,28 @@ const ContentMessageComponent: React.FC<ContentMessageProps> = ({
   const reactionGroupedByUser = groupByReactionUsers(reactions);
   const reactionsTotalCount = Array.from(reactionGroupedByUser).length;
 
+  const isUserRemovingReaction = (reaction: ReactionType): boolean => {
+    const reactions = message.reactions();
+    const userId = selfId.id;
+    const userReactions = reactions[userId] || '';
+
+    if (userReactions) {
+      const reactionsArr = userReactions.split(',');
+      const reactionIndex = reactionsArr.indexOf(reaction);
+      return reactionIndex !== -1;
+    }
+    // first time reacted
+    return false;
+  };
+  const [selectedEmojiurl, setSelected] = useState('');
```

Comment:
Let's uppercase `Url` and name a setter `setSelectedEmojiUrl` for consistency

Reply:
And maybe let's use `null` as initial/empty value, this way we're sure in the code if we're dealing with emoji or empty value.

Reply:
we are dealing with a string here, i will initialise it with empty string